src/xFactor/Application.java

```diff
@@ -0,0 +1,204 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Set;
+
+import xFactor.Person.Gender;
+
+public class Application {
+	private ArrayList<Judge> allJudges;
+	private ArrayList<Judge> judges;
+	private ArrayList<Participant> allParticipants;
+	private ArrayList<Participant> participants;
+	private Stage[] allStages;
+	private ArrayList<String> qualities;
+	
+	private PersonServiceImpl personService = new PersonServiceImpl();
+	private StageServiceImpl stageService = new StageServiceImpl();
+	private Competition competition = new Competition();
+			
+	private Random random = new Random();
+	
+	public Stage getStage(int stageNum) {
+		return allStages[stageNum - 1];
+	}
+	
+	private void addStages() {

```

Comment:
Why you need this method, when you pass null as value you won't save the reference? do you want to clean the array?
Also why are you using array instead of list?

Reply:
So that I can get a stage by passing its number (which is its index + 1) to the method getStage(int number).